code/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './Header.css'
+
+export const Header = () => {
```

Comment:
Even though you haven't implemented whole logic for `<Header />` as we tried to, congratulations for trying :)